ledger/src/structure.rs

```diff
@@ -759,6 +759,9 @@ pub struct Intent<S: SignatureKind<D>, P: ProofKind<D>, B: Storable<D>, D: DB> {
 }
 tag_enforcement_test!(Intent<(), (), Pedersen, InMemoryDB>);
 
+// Intent with a PedersenRandomness
+pub(crate) type PedersenRandomnessIntent<S, P, D> = Intent<S, P, PedersenRandomness, D>;
```

Comment:
I'm not really a fan of this. It doesn't actually simplify the type, just shuffles around the wording and introduces indirection. I believe in most cases here we also use `P = ProofPreimageMarker`, so maybe an alias like `type UnprovenIntent<S, D> = Intent<S, ProofPreimageMarker, PedersenRandomness, D>` makes more sense?